test/16_recode.jl

```diff
@@ -14,7 +14,6 @@ const ≅ = isequal
     x in ([1:10;], CategoricalArray(1:10), CategoricalArray{Union{Int, Missing}}(1:10)),
     y in (similar(x), Array{Int}(size(x)),
           CategoricalArray{Int}(size(x)), CategoricalArray{Union{Int, Missing}}(size(x)), x)
-
```

Comment:
OK, the package doesn't appear to be using a completely consistent style. In general PRs should adopt whatever style is used in a given file, with the goal of keeping them as consistent as possible and limiting the diff size. Here, better not remove these empty lines, and use the same style below (but not in the other file).

Reply:
I have put a newline after every multiline for